client/css/widgets/scoreboard.css

```diff
@@ -0,0 +1,66 @@
+.scoreboard > div.scoreboardIntermediate {
+  position: relative;
+  z-index: 0;
+  border: 2px solid black;
+  box-sizing: border-box;
+  background: #fff;
+  border-radius: 8px;
+  overflow: auto;
+  height: 100%;
+  width: 100%;
+}
+
+.scoreboard > div.scoreboardIntermediate > table {
+  border-collapse: separate;
+  border-spacing: 0;
+  width: 100%;
+}
+
+.scoreboard > div.scoreboardIntermediate > table tbody td {
+  border-bottom: 1px solid #ddd;
+  border-right: 1px solid #ddd;
+  color: #333;
```

Comment:
Similar to my argument for why background-color should only be specified on the root element unless different colors are needed, I think this color should also only be specified on the root so that a developer can define their own general color scheme with e.g. `css: "background-color: #00f; color: #ff0;"` applied to the scoreboard.